src/main/java/de/muenchen/isi/api/dto/stammdaten/LookupListsDto.java

```diff
@@ -42,4 +42,6 @@ public class LookupListsDto {
     private LookupListDto artGsNachmittagBetreuung;
 
     private LookupListDto sobonOrientierungswertJahr;
+
+    private LookupListDto sobonOrientierungswertJahrWithoutStandortabfrage;
```

Comment:
Da die meisten Variablen auch eher eine Zusammensetzung deutscher Wörter sind, würde ich das hier ebenfalls so machen.
```suggestion
    private LookupListDto sobonOrientierungswertJahrOhneStandortabfrage;
```

Reply:
In den Coding-Guidelines ist definiert, dass fachliche Benennungen in Deutsch und der Rest in Englisch geschrieben wird.